src/components/pages/InstructorNewsFeed/NewsContainer.js

```diff
@@ -2,41 +2,26 @@ import React, { useEffect } from 'react';
 import {connect} from 'react-redux';
 import '../../../styles/index.less';
 import {getNewsFeeds} from '../../../redux/actions/instructorActions';
+import IndividualNews from './IndividualNews';
 function NewsContainer(props) {
   const { setPostId, setPostOptions, newsfeed, dispatch }=props;
   useEffect(() => {
-    const token = JSON.parse(localStorage.getItem('okta-token-storage'));
-    const config = {
-      headers: { Authorization: `Bearer ${token.idToken.value}` },
-    };
-    dispatch(getNewsFeeds(config));
```

Comment:
Removing the token authentication cleans up the code and using the axiosWithAuth from utils will make everything  easy to read.